### ISSUE TITLE
[BUG]: user and group module does not support specifying user_id/group_id at creation

### ISSUE THREAD

**Issue by sigbjornaib** [dell/ansible-powerscale#48]
**Describe the bug**
When creating a user or group where both the user_name and user_id for a new user, or group_name and group_id for a new group, an error is thrown: 
```
"parameters are mutually exclusive: user_name|user_id"
```

Specifying a uidnumber or gidnumber when creating users and groups using the Isilon CLI is supported and works. 
```
isi auth groups add group1000 --gid 1000 --zone myzone --provider local
```

**To Reproduce**
1. Write a task using dellemc.powerscale.user or dellemc.powerscale.group
2. Enter a group_name/user_name in the task
3. Enther a group_id/user_id in the same task.
...
n. Step n See error
```
      - name: Create/Modify Local Groups
        dellemc.powerscale.group:
          onefs_host: "{{ onefs_hostname }}"
          port_no: "{{ onefs_port }}"
          verify_ssl: "{{ onefs_verify_ssl }}"
          api_user: "{{ api_user }}"
          api_password: "{{ api_password }}"
          group_name: "group{{ item.gidnumber }}"
          group_id: "{{ item.gidnumber }}"
          access_zone: "{{ item.access_zone }}"
          provider_type: "local"
          state: "{{ item.state | default('present') }}"
        loop: "{{ isilon_groups }}"
```

**Expected behavior**
We need to create local users for legacy systems with specific uidnumbers/gidnumbers. We have automated almost all the Isilon/Powerscale setup using Ansible. Not being able to create users and groups is a setback for a fully automated Isilon/Powerscale setup.

**Screenshots**

**Logs**
If applicable, submit logs or stack traces from the affected services

**System Information (please complete the following information):**
 - OS/Version: RHEL8
 - Ansible Version core 2.13.6
 - Python Version 3.10

**Comment by anupamaloke** [comment]
@sigbjornaib, thank you for reporting this issue. This has been added to the v1.9.0 milestone.

**Comment by anupamaloke** [comment]
Fixed by #51
